ui/src/components/member/AddPoc.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright The Athenz Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import AddModal from '../modal/AddModal';
+import MemberUtils from '../utils/MemberUtils';
+import styled from '@emotion/styled';
+import InputLabel from '../denali/InputLabel';
+import InputDropdown from '../denali/InputDropdown';
+import { colors } from '../denali/styles';
+import { selectAllUsers } from '../../redux/selectors/user';
+import { USER_DOMAIN } from '../constants/constants';
+import RequestUtils from '../utils/RequestUtils';
+
+const SectionsDiv = styled.div`
+    width: 760px;
```

Comment:
Would `max-width` make it responsive?

Reply:
the styling properties are copied from add member modal, just tested the AddPoc modal is responsive to window resize.